### PR TITLE
WSTEAM1-780: Merge NWS-1662 into latest

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves [WSTEAM1-780](https://jira.dev.bbc.co.uk/browse/WSTEAM1-780)

Overall changes
======
Replace references to the **/news/** iChef path to the new **/ace/ws/** wherever the buildIChefUrl function is used. Update tests, fixture data and snapshots.

Code changes
======
- Update buildiChefUrl function
- Update iChef URLs in tests snapshots, fixture data and stories that utilise the buildiChefUrl function.
- Refactor buildiChefUrl function to not add PNG parameter. Refactor byline extractor to remove redundent props.
- **Note that there is a getBrandedImage function that still uses /news/**.

Testing
======
- Pull down PR and search there are no instances of ichef.bbci.co.uk/news/ or ichef.test.bbci.co.uk/news/ (other than those related to the getBrandedImage function).
- Check through Storybook to ensure no broken images
- Check through local pages to ensure no broken images (including jpg, PNG, webp)
- On test - check Byline image renders as expected (unable to find a local example) - https://www.test.bbc.com/pidgin/articles/c39g87n9jp0o

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/components/Byline/utilities/bylineExtractor/index.tsx [hunk 1]
```diff
@@ -55,7 +55,6 @@ const bylineExtractor = (blocks: object[]) => {
     originCode,
     locator,
     resolution: 160,
-    isPng: true,
```
Comment (karinathomasbbc) [inline src/app/components/Byline/utilities/bylineExtractor/index.tsx:58]: Is this no longer required?

File: src/app/components/Curation/HierarchicalGrid/liveFixtures.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const livePromo = [
     lastPublished: '2022-11-13T13:34:32.000Z',
     link: 'https://www.bbc.com/pidgin/live/56000740',
     imageUrl:
-      'https://ichef.bbci.co.uk/news/{width}/cpsprodpb/C6E6/production/_116881905_trump.jpg',
+      'https://ichef.bbci.co.uk/ace/ws/{width}/cpsprodpb/C6E6/production/_116881905_trump.jpg',
```
Comment (karinathomasbbc) [inline src/app/components/Curation/HierarchicalGrid/liveFixtures.js:9]: What's the difference between `ace/ws` and `ace/standard`?

**Review by karinathomasbbc — APPROVED** [review]

**Review by andrewscfc — COMMENTED** [review]
Hey, am I misunderstanding something with this PR, shouldn't we be using `ace/ws` everywhere so we can customise the recipe to our needs? `ace/standard` is used more widely so we couldn't customise it

**Review by Isabella-Mitchell — COMMENTED** [review]
Ah yes second to Andrew's comment - I can see this PR I worked on when we thought we would use the **ace/standard/** path. The task in this ticket https://jira.dev.bbc.co.uk/browse/WSTEAM1-780 is to use the **ace/ws** path. So this PR is nearly there now latest has been merged in, just the paths and the PR description need updating

File: src/app/components/Byline/utilities/bylineExtractor/index.tsx (hunk 1 above)
Comment (shayneahchoon) [reply src/app/components/Byline/utilities/bylineExtractor/index.tsx:58]: Heya, by the looks of it, it's no longer used.

**Review by Isabella-Mitchell — APPROVED** [review]
Looks good! Just flagging that we might want to let iChef know ahead of merging in case we increase the traffic to iChef servers.

**Pull request merged**